lib/matplotlib/tri/triinterpolate.py

```diff
@@ -287,7 +286,7 @@ def _interpolate_single_key(self, return_key, tri_index, x, y):
 
 class CubicTriInterpolator(TriInterpolator):
     r"""
-    A CubicTriInterpolator performs cubic interpolation on triangular grids.
+    Cubic interpolator on triangular grids.
```

Comment:
```suggestion
    Cubic interpolator on a triangular grid.
```
Like in `LinearTriInterpolator` above.